rules-tests/Php71/Rector/ClassConst/PublicConstantVisibilityRector/Fixture/public_visibility_final_const.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\Php71\Rector\ClassConst\PublicConstantVisibilityRector\Fixture;
+
+class PublicVisibilityFinalConstant
+{
+    const EXIT_OK = 0;
+    final const EXIT_KO = 1;
+    final const EXIT_NULL = 2;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php71\Rector\ClassConst\PublicConstantVisibilityRector\Fixture;
+
+class PublicVisibilityFinalConstant
+{
+    public const EXIT_OK = 0;
```

Comment:
this one need to be removed as unrelated, only change the `final` constant part:

```suggestion
```

Reply:
this one too :)